src/components/analytics/metrics-table.vue

```diff
@@ -113,7 +113,14 @@ export default {
       "num_followup_forms": "Number of Forms that use the Entity List",
       "num_entities": "Number of Entities in the Entity List",
       "num_failed_entities": "Number of Entity creation errors",
-      "num_entity_updates": "Number of Entity updates"
+      "num_entity_updates": "Number of Entity updates",
+      "sso_enabled": "SSO enabled on Server",
```

Comment:
What do you think about not capitalizing "Server", since it's not a Central term?